test/manager.js

```diff
@@ -1132,6 +1132,58 @@ describe('Manager', () => {
             expect(rendered).to.contain('<h1>default message</h1>');
         });
 
+        it('renders with an async global context function (no request)', async () => {
+
+            const asyncFn = () => Promise.resolve('from async');
```

Comment:
```suggestion
            const asyncFn = async () => Promise.resolve('from async');
```

Reply:
Just cuz it works, and cuz you can return whatever value you'd like now instead of Promise.resolve(xyz)

Reply:
`await` is needed there for the linter

Reply:
I'll update these tonight